version.c

```diff
@@ -35,21 +35,8 @@
 #include <sys/utsname.h>
 #include <unistd.h>
 #include "mutt/mutt.h"
+#include "mutt/idna2.h"
```

Comment:
We don't need this.
It's already included by `#include <mutt/mutt.h>`

Reply:
Yes, i just want to tell you this.